package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "webpack --mode=production",
     "prestart": "yarn",
-    "start": "webpack-dev-server --hot --mode=development --host 0.0.0.0 --disableHostCheck=true",
+    "start": "webpack-dev-server --hot --mode=development --host 0.0.0.0 --port 9966 --disableHostCheck=true",
```

Comment:
Any reason for changing this port?

Reply:
Yea, I changed it so that when we're running locally we don't have any issues with Auth0 expecting things on a different port.